src/Common/src/System/Net/Internals/IPAddressExtensions.cs

```diff
@@ -18,7 +18,16 @@ public static IPAddress Snapshot(this IPAddress original)
 #pragma warning restore CS0618
 
                 case AddressFamily.InterNetworkV6:
-                    return new IPAddress(original.GetAddressBytes(), (uint)original.ScopeId);
+                    Span<byte> addressBytes;
+                    unsafe
+                    {
+                        // TODO https://github.com/dotnet/roslyn/issues/17287: Clean up once we can stackalloc into a span
```

Comment:
This issue appears to be for readonly refs

Reply:
About two-thirds of the way down the issue:
"syntax for stackalloc’d spans
I think the most popular proposal is target typing - “Span sp = stackalloc int[100];” We should discuss/agree on this."